spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -181,9 +181,9 @@
               caffeinated.unassign(decaffeinated)
             end
 
-            expect(user.connected?(decaffeinated)).to be true
-            expect(user.connected?(caffeinated)).to be true
-            expect(caffeinated.connected?(decaffeinated)).to be false
+            expect(user.connected?(decaffeinated)).to be_truthy
```

Comment:
By hand. And no, those aren't the same. All cases of `be_true` should now be `be_truthy`, `be true` should be unchanged.